Tests/SwiftAlgorithmsTests/TrimTests.swift

```diff
@@ -56,4 +56,14 @@ final class TrimTests: XCTestCase {
     let results_one = [2, 10, 12, 15, 20, 100].trimming { $0.isMultiple(of: 2) }
     XCTAssertEqual(results_one, [15])
   }
+    
```

Comment:
Could we add unit test cases for `trim`?

Reply:
Yes, I added a checkbox in the PR description. I was just waiting for a review before adding those.